spec/system/course_progress_badge_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'Course Progress Badge', type: :system do
+  let!(:path) { create(:path, default_path: true) }
+  let!(:course) { create(:course, path: path) }
+  let!(:section) { create(:section, course: course) }
+  let!(:first_lesson) { create(:lesson, section: section) }
+  let!(:second_lesson) { create(:lesson, section: section) }
+  let!(:user) { create(:user) }
+
+  before do
+    sign_in(user)
+  end
+
+  context 'when course has not been started' do
+    it 'does not show progress percentage' do
+      visit path_course_path(path, course)
+
+      within find(:test_id, 'default-badge') do
+        expect(page).not_to have_content('0%')
+        expect(page).not_to have_content('COMPLETE')
```

Comment:
Just a nitpick, I think we could probably get away with not asserting for the "Complete" copy on the page. The reason I bring it up is because it threw me a little when I first read it, I thought "Complete" might have only been a label that appeared when the course was fully complete 😆 

Whats your thoughts?

Reply:
I'm totally fine with removing 'COMPLETE' as it is redundant test and it can cause confusion.